corehq/apps/callcenter/utils.py

```diff
@@ -118,18 +119,22 @@ def valid_element_name(name):
 def sync_call_center_user_case(user):
     domain = user.project
     if domain and domain.call_center_config.enabled:
-        owner_id = call_center_case_owner(user, domain.call_center_config)
-        sync_user_case(user, domain.call_center_config.case_type, owner_id)
+        owner_id, case = call_center_case_and_owner(user, domain)
+        sync_user_case(user, domain.call_center_config.case_type, owner_id, case)
 
 
-def call_center_case_owner(user, config):
+def call_center_case_and_owner(user, domain):
```

Comment:
i might suggest a few possible changes to this method name

1. Mark it private to the module (so starting with an `_`) so that future editors can refactor without worrying about breaking things outside this module
2. Make the method name a verb (so `get_call_center_case_and_owner` or maybe `get_call_center_case_and_default_owner`)
3. Make the order of the case/owner consistent with the return value.